### ISSUE TITLE
Rows tab no longer works on tables with uppercase characters

### ISSUE THREAD

**Issue by transcranial** [sosedoff/pgweb#412]
Rows tab was working in 0.10.0. Broke in 0.11.0 for tables with uppercase characters in its name. Other tabs as well as "Table Information" continue to work fine.

For example, on a table named "SequelizeMeta" (created by [sequelize](https://github.com/sequelize/sequelize/blob/master/docs/migrations.md) for migrations):

![image](https://user-images.githubusercontent.com/6182852/52424383-e926bf80-2ac7-11e9-83bb-27cb3f70690b.png)

**Comment by sosedoff** [comment]
Can't reproduce in 0.11.1 (latest) on OSX and with Heroku test database. Need more info: OS, Postgres version.

**Comment by transcranial** [comment]
Using docker compose:

```yml
version: '3'
services:
  postgres:
    image: postgres:9.6-alpine
    ports:
      - 5432:5432
  pgweb:
    restart: always
    image: sosedoff/pgweb:0.11.1
    ports:
      - 8081:8081
    links:
      - postgres:postgres
    environment:
      - DATABASE_URL=postgres://postgres:postgres@postgres:5432/postgres?sslmode=disable
    depends_on:
      - postgres
```

Create table using:
```sql
CREATE TABLE "testTable" (
  id    integer PRIMARY KEY,
  name   varchar(40)
);
```

Click on "Rows" tab, get following message:

`ERROR: pq: relation "public.testtable" does not exist`

Also tried combos of:
`postgres:10.6-alpine`
`postgres:11.1-alpine` 
and 
`sosedoff/pgweb:0.11.0`
`sosedoff/pgweb:0.10.0`. 

"Rows" tab works fine on `sosedoff/pgweb:0.10.0`.

**Comment by sosedoff** [comment]
Alright, thanks for the info. I'll try to fix this today.

**Comment by sosedoff** [comment]
Fixed in 0.11.2.
